README.md

```diff
@@ -295,6 +296,56 @@ clip-retrieval-front 3005
 
 You can also run it with `clip-retrieval front` or back from the python package.
 
+## Clip client
```

Comment:
Let's put this higher in the readme, like line 35 as I figure lot of people will want to use it